fixed card component

djek195 opened the pull request:
1. Changed size product image and hover
2. Added limit of showing product name
3. Added hint when the user hovered the mouse over the product name.

djek195 committed:
fixed card component

1. Changed size product image and hover
2. Added limit of showing product name
3. Added hint when the user hovered the mouse over the product name.

djek195 committed:
Update card.module.css

dimalian commented on src/components/Card/Card.js:
```diff
@@ -3,18 +3,27 @@ import { Link } from 'react-router-dom';
 
 const Card = ({ product }) => {
-  const { href, title, image, alt, price } = product;
+  const { href, title, image, price } = product;
+
+  let maxTitleLength = 15;
+  if (window.innerWidth >= 630 && window.innerWidth <= 1023) {
+    maxTitleLength = 12;
+  } else if (window.innerWidth >= 1023) {
+    maxTitleLength = 19;
+  }
+
+  const trimmedTitle = title.length > maxTitleLength ? `${title.slice(0, maxTitleLength)}...` : title;
```
you can do it much simpler and clearly by using `text-overflow: ellipsis;` style. https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow

djek195 replied:
Done

djek195 committed:
updated card styles

Changes made with included comments in the PR
1. Deleted JS logic
2. Added CSS property for hidden overflow content

djek195 committed:
Merge branch 'fix-card-component' of https://github.com/store-challenge/store-app into fix-card-component
